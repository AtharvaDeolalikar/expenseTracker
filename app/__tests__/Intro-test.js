import React from "react";
import renderer from "react-test-renderer";
import Transanctions from "../src/components/expensesList";

test("renders correctly", () => {
  const tree = renderer.create(<Transanctions />).toJSON();
  expect(tree).toMatchSnapshot();
});
