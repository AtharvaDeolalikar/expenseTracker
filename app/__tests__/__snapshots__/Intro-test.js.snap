// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
        "fontWeight": "400",
        "marginBottom": 10,
      }
    }
  >
    All Expenses
  </Text>
  <RCTScrollView
    getItem={[Function]}
    getItemCount={[Function]}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    style={
      Object {
        "flex": 1,
      }
    }
    viewabilityConfigCallbackPairs={Array []}
  >
    <View />
  </RCTScrollView>
</View>
`;
